lavamoat/browserify/policy-override.json

```diff
@@ -58,6 +58,11 @@
         "XMLHttpRequest": true
       }
     },
+    "@metamask/controllers>web3": {
+      "globals": {
+        "XMLHttpRequest": true
```

Comment:
I've been having issues with this flipping to "write". Meant to add the override myself, but still don't understand what this is about... @Gudahtt can you explain?

Reply:
I cannot. See this commit message:

>The `web3` package used by `@metamask/controllers` unintentionally
overwrites the `XMLHttpRequest` global, which breaks things. This was
fixed by revoking `web3`'s write access to that global using a policy
override.
>
>Previously this policy override was applied to `web3`, but for some
unknown reason, this update caused that override to no longer apply.

There is a thread in the LavaMoat channel about it. I'd recommend following that thread for further updates. We haven't yet identified the problem.